libparsec/src/handle.rs

```diff
@@ -9,6 +9,7 @@ use libparsec_client::{
 use libparsec_platform_async::event::Event;
 use libparsec_types::prelude::*;
 
+/// Used to identify client handle
```

Comment:
Handle are also used to identify running workspace, mountpoint, and invitation progress (see the `HandleItem` enum